ansible/deploy.py

```diff
@@ -0,0 +1,162 @@
+import click
+import git
+import os
+import subprocess
+import requests
+import json
+
+
+@click.command()
+@click.option('--host', required=True, type=click.Choice(['staging', 'production']), help='The host to deploy to')
```

Comment:
Suggestion: if you add an alias `--to` you can then do `python deploy.py --to production`, which might be a little nicer / easier to remember than `python deploy.py --host production`.

```suggestion
@click.option('--host', '--to', required=True, type=click.Choice(['staging', 'production']), help='The host to deploy to')
```

Reply:
I love this one!